libraries/testing/src/secrets.js

```diff
@@ -36,7 +36,7 @@ class Secrets {
           // delay loading cfg until we know we need it (allowing this to work with
           // loaders that do not have a `cfg` component); note that this also ensures
           // later calls to load.cfg(..) will work.
-          if (!cfg) {
+          if (!cfg && this.load) {
```

Comment:
This makes mockSuite able to operate without stickyLoader, right?  If so, that should get a docs update in the README.